homeassistant/components/iotawatt/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "iotawatt",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "name": "[%key:common::config_flow::data::name%]",
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      },
+      "auth": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "description": "The IoTawatt device requires authentication. Please enter the username and password and click the Submit button."
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
We don't set a unique id for the config entry and we don't use this abort reason.

Reply:
If possible it's good to set a unique id for the config entry.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-id-requirements

Reply:
Removed the ```abort``` section from the ```strings.json``` file.